sdk/java/pulumi/src/main/java/com/pulumi/experimental/automation/serialization/internal/Model.java

```diff
@@ -0,0 +1,7 @@
+// Copyright 2025, Pulumi Corporation
+
+package com.pulumi.experimental.automation.serialization.internal;
+
+interface Model<T> {
+    T convert();
+}
```

Comment:
Is this interface actually consumed anywhere? Sorry if I missed it.

Reply:
I added a comment to explain. It's only really used so that internal serialization model classes reference the public classes, so that when you're working on the implementation and use "Find All References" on the public class, it'll point you to the model classes.